packages/components/addon/components/hds/breadcrumb/item.hbs

```diff
@@ -13,18 +13,36 @@
       <span class="hds-breadcrumb__text">{{@text}}</span>
     </div>
   {{else}}
-    <LinkTo
-      class="hds-breadcrumb__link"
-      @models={{hds-link-to-models @model @models}}
-      @query={{hds-link-to-query @query}}
-      @route={{@route}}
-    >
-      {{#if @icon}}
-        <div class="hds-breadcrumb__icon">
-          <FlightIcon @name={{@icon}} @size="16" @stretched={{true}} />
-        </div>
-      {{/if}}
-      <span class="hds-breadcrumb__text">{{@text}}</span>
-    </LinkTo>
+    {{#if @isRouteExternal}}
+      <LinkToExternal
+        class="hds-breadcrumb__link"
+        @current-when={{@current-when}}
+        @models={{hds-link-to-models @model @models}}
+        @query={{hds-link-to-query @query}}
+        @replace={{@replace}}
+        @route={{@route}}
+      >
+        {{#if @icon}}
+          <div class="hds-breadcrumb__icon">
+            <FlightIcon @name={{@icon}} @size="16" @stretched={{true}} />
+          </div>
+        {{/if}}
+        <span class="hds-breadcrumb__text">{{@text}}</span>
+      </LinkToExternal>
+    {{else}}
+      <LinkTo
+        class="hds-breadcrumb__link"
+        @models={{hds-link-to-models @model @models}}
+        @query={{hds-link-to-query @query}}
+        @route={{@route}}
```

Comment:
since the `<LinkToExternal>` exposes the `@current-when` and `@replace` arguments` we should add them to the `<LinkTo>` case too, for consistency (and ease in documenting the API).

```suggestion
        @current-when={{@current-when}}
        @models={{hds-link-to-models @model @models}}
        @query={{hds-link-to-query @query}}
        @replace={{@replace}}
        @route={{@route}}
```